constants.py

```diff
@@ -0,0 +1,8 @@
+WOLT_URL = "https://consumer-api.wolt.com/order-xp/web/v1/venue/slug/<RESTAURANT_ID>/dynamic/?lat=<LATITUDE>&lon=<LONGITUDE>"
```

Comment:
Use f-strings. Look at my other comment about this in the wolt_services.py file.